.github/workflows/formatter.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 7.0.x
+          dotnet-version: 8.0.x
```

Comment:
I think if you drop the `dotnet-version` it takes the one found from the global.json

Reply:
Oh, indeed it works. I didn't know that trick. Thanks!